sklearn/tests/test_common.py

```diff
@@ -268,3 +270,56 @@ def test_strict_mode_check_estimator():
 def test_strict_mode_parametrize_with_checks(estimator, check):
     # Ideally we should assert that the strict checks are Xfailed...
     check(estimator)
+
+
+column_name_modules_to_ignore = {
```

Comment:
This is the ignore list. When more modules get added, we can remove it from this list to make sure it gets tested. After we finish each module we can move the checks into `check_estimator`.